packages/material-ui-icons/builder.js

```diff
@@ -90,27 +90,10 @@ async function generateIndex(options) {
   await fse.writeFile(path.join(options.outputDir, 'index.js'), index);
 }
 
-async function worker({ svgPath, options, renameFilter, template }) {
-  process.stdout.write('.');
-
-  const normalizedSvgPath = path.normalize(svgPath);
-  const svgPathObj = path.parse(normalizedSvgPath);
-  const innerPath = path
-    .dirname(normalizedSvgPath)
-    .replace(options.svgDir, '')
-    .replace(path.relative(process.cwd(), options.svgDir), ''); // for relative dirs
-  const destPath = renameFilter(svgPathObj, innerPath, options);
-
-  const outputFileDir = path.dirname(path.join(options.outputDir, destPath));
-  const exists2 = await fse.exists(outputFileDir);
-
-  if (!exists2) {
-    console.log(`Making dir: ${outputFileDir}`);
-    mkdirp.sync(outputFileDir);
-  }
-
-  const data = await fse.readFile(svgPath, { encoding: 'utf8' });
+// Noise introduced by Google by mistake
```

Comment:
Did google acknowledge that or is there maybe a deeper cause for this issue?

Reply:
Google has almost abandoned their icons repository: https://github.com/google/material-design-icons. Considering how many issues are opened without any answer, I fear that we wouldn't have any acknowledgment from Google. It's a point I will mention to them this afternoon (during the one hour call with a Google Material advocate).

Reply:
From my perspective, the [diff](https://user-images.githubusercontent.com/3165635/59632182-ec9f1700-9149-11e9-9ae2-ea032ceebaa2.png) is good enough for moving forward. But yeah, if would be awesome if they fix it upstream!

Reply:
Yeah that works for me. We just need to keep an eye on it. Not that we're hiding a more fundamental issue with some app setups.